### PR TITLE
feat: reordering items overview

### PR DESCRIPTION
**Pull request by sahas-01**

### CODE REVIEW

**Review by KanishkaRajputd — COMMENTED** [review]
wrong values found . please be very care full regarding any single value

File: src/global/post_trackInfo.ts [hunk 1]
```diff
@@ -38,43 +38,72 @@ export const networkTrackInfo: INetworkTrackInfo = {
...
-			'maxDeciding': 100,
-			'decisionDeposit': 333333333330000,
-			'preparePeriod': 300,
+		[PostOrigin.BIG_SPENDER] : {
+			'trackId': 34,
+			'description': 'Origin able to spend up to 33,333 KSM from the treasury at once.',
+			'group': 'Treasury',
+			'name': 'big_spender',
+			'maxSpend': 33333333,
+			'maxDeciding': 50,
```
Comment (KanishkaRajputd) [inline src/global/post_trackInfo.ts:47]: wrong value for maxSpend

File: src/global/post_trackInfo.ts [hunk 2]
```diff
@@ -603,83 +603,245 @@ export const networkTrackInfo: INetworkTrackInfo = {
...
+		[PostOrigin.BIG_SPENDER] : {
+			'trackId': 34,
+			'description': 'Origin able to spend up to 1,000,000 DOT from the treasury at once',
+			'group': 'Treasury',
+			'name': 'big_spender',
+			'maxSpend': 1000000,
+			'maxDeciding': 50,
+			'decisionDeposit': 4000000000000,
+			'preparePeriod': 2400,
+			'decisionPeriod': 403200,
```
Comment (KanishkaRajputd) [inline src/global/post_trackInfo.ts:708]: wrong decision period

**Review by KanishkaRajputd — COMMENTED** [review]
please resolve comments

File: src/global/post_trackInfo.ts (hunk 1 above)
Comment (sahas-01) [reply src/global/post_trackInfo.ts:47]: fixed, pls check

**Pull request merged**